.circleci/config.yml

```diff
@@ -189,7 +189,9 @@ jobs:
             pip install --upgrade -e /usr/src/artman/
       - run:
           name: Build Pub/Sub (Java)
-          command: artman --local --config=/googleapis/google/pubsub/artman_pubsub.yaml publish --dry-run --github-username foo --github-token bar --target staging java_gapic
+          command: |
+            git clone https://github.com/googleapis/api-client-staging /tmp/api-client-staging
+            artman --local --config=/googleapis/google/pubsub/artman_pubsub.yaml publish --dry-run --local-repo-dir=/tmp/api-client-staging --github-username foo --github-token bar --target staging java_gapic
```

Comment:
I'm wondering whether it would be more informative to call the current semantics of `--dry_-run` something like `--no_push` instead, meaning we're not pushing any generated code to any place outside the local machine (regardless of what the publishing type is, GITHUB or something else in the future).

Then we could use `--dry-run` in its more conventional meaning: echoing the commands/processes/tasks artman would run, but without actually running them.

WDYT? Obviously it would be simpler to do this in two follow-up CLs, one for the flag rename and one for the new dry run functionality.

Reply:
I saw your point. Currently `--dry-run` is a subflag of `publish` command. It means do everything except publish (or push). So it is equivalent to `--no_push`. I got the name from some internal usage, where a tool prepares the config (including merging and normalization) without real pushing. I think we won't have `--no-push` and `--dry-run` for two different behaviors. Between those two, I personally prefer the latter.

Reply:
I guess what I'm trying to say is that I've seen `--dry-run` conventionally refer to printing the commands and configs that would be executed, so in this case it would print `git push ....` and implies a `--no_push`. So maybe you're right , we could do without `--no_push` if you make `--dry-run` specify the commands it's not running.

Reply:
Sounds good. I am open to flag name change.